core/arch/arm/plat-stm/platform_config.h

```diff
@@ -190,132 +177,37 @@
 #define SCU_CTRL_INIT			0x00000065
 
 /*
- * TEE RAM layout without CFG_WITH_PAGER:
- *
- *  +---------------------------------------+  <- CFG_DDR_TEETZ_RESERVED_START
- *  | TEE private secure |  TEE_RAM         |   ^
- *  |   external memory  +------------------+   |
- *  |                    |  TA_RAM          |   |
- *  +---------------------------------------+   | CFG_DDR_TEETZ_RESERVED_SIZE
- *  | Secure Data Path test memory (opt.)   |   |
- *  +---------------------------------------+   |
- *  |     Non secure     |  SHM             |   |
- *  |   shared memory    |                  |   |
- *  +---------------------------------------+   v
- *
- *  TEE_RAM : default 1MByte
- *  TA_RAM  : all what is left
- *  SDP_RAM : optional default SDP test memory 8MByte
- *  PUB_RAM : default 2MByte
- *
- * ----------------------------------------------------------------------------
- * TEE RAM layout with CFG_WITH_PAGER=y:
- *
- *  +---------------------------------------+  <- CFG_CORE_TZSRAM_EMUL_START
- *  | TEE private highly | TEE_RAM          |   ^
- *  |   secure memory    |                  |   | CFG_CORE_TZSRAM_EMUL_SIZE
- *  +---------------------------------------+   v
- *
- *  +---------------------------------------+  <- CFG_DDR_TEETZ_RESERVED_START
- *  | TEE private secure |  TA_RAM          |   ^
- *  |   external memory  |                  |   |
- *  +---------------------------------------+   | CFG_DDR_TEETZ_RESERVED_SIZE
- *  | Secure Data Path test memory (opt.)   |   |
- *  +---------------------------------------+   |
- *  |     Non secure     |  SHM             |   |
- *  |   shared memory    |                  |   |
- *  +---------------------------------------+   v
- *
- *  TEE_RAM : default 256kByte
- *  TA_RAM  : all what is left in DDR TEE reserved area
- *  SDP_RAM : optional default SDP test memory 8MByte
- *  PUB_RAM : default 2MByte
+ * Register nsecure ddr chunks for dynamic shm: these are DDR ranges
```

Comment:
s/nsecure/non-secure/
s/ddr/DDR/
Also in comments, I think it is nice to read "shared memory" instead of just "shm".

Reply:
thanks.